flow/util/genMetrics.py

```diff
@@ -349,24 +349,24 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
                        logPath + '/2_1_floorplan.log',
                        occurrence=0)
 
-    extractTagFromFile('floorplan__design__instance__stdcell__area',
+    extractTagFromFile('floorplan__design__instance__area__stdcell',
                        metrics_dict,
                        baseRegEx.format('floorplan final report_design_area',
                                         '^Design area (\S+) u\^2'),
                        logPath + '/2_1_floorplan.log')
 
-    extractTagFromFile('floorplan__design__instance__design__util',
+    extractTagFromFile('floorplan__design__instance__utilization',
                        metrics_dict,
                        baseRegEx.format('floorplan final report_design_area',
                                         '^Design area .* (\S+)% utilization'),
                        logPath + '/2_1_floorplan.log')
 
-    extractTagFromFile('floorplan__design__io__count',
+    extractTagFromFile('floorplan__design__io',
```

Comment:
why drop count?

Reply:
We had removed the modifier "count"  when it was only entity.  So we now have "count" and "area" only for instances/macros etc.